src/docfx/cli/Program.cs

```diff
@@ -94,6 +94,8 @@ private static (string command, string docset, CommandLineOptions options) Parse
                 syntax.DefineOption("o|output", ref options.Output, "Output directory in which to place built artifacts.");
                 syntax.DefineOption("log", ref options.Log, "Output build log path.");
                 syntax.DefineOption("legacy", ref options.Legacy, "Enable legacy output for backward compatibility.");
+                syntax.DefineOption("repo", ref options.Repo, "Overrides the current git repo name guessed from remote URL.");
+                syntax.DefineOption("branch", ref options.Branch, "Overrides the current git branch name guessed.");
```

Comment:
these two options are used for overwrite configuration config? what about contribution-repo and contribution-branch